builtin/settingtypes.txt

```diff
@@ -256,8 +256,8 @@ fps_max (Maximum FPS) int 60 1 4294967295
 #    Vertical screen synchronization. Your system may still force VSync on even if this is disabled.
 vsync (VSync) bool false
 
-#    Maximum FPS when the window is not focused, or when the game is paused.
-fps_max_unfocused (FPS when unfocused or paused) int 20 1 4294967295
+#    Maximum FPS when the window is not focused.
+fps_max_unfocused (FPS when unfocused) int 10 1 4294967295
```

Comment:
note that this is a string change and can't go into 5.11.0, if we want that.

Reply:
Then the options are:
* Don't merge any of this into 5.11.0.
* Only merge <https://github.com/luanti-org/luanti/pull/15774/commits/3a0d35d775d5dc5e6fb41f12852e23f911dbe8bd>.
* Merge with wrong setting description.


Btw. in case of a pre release merge, would minetest.conf.example need to be updated for the default to 10 change?

Reply:
> Btw. in case of a pre release merge, would minetest.conf.example need to be updated for the default to 10 change?

changing defaults is fine, just not the description. would make sense to consistently update it in that case.
either way option 2 sounds fine.